### DISCUSSION TITLE
Colors for Numeric Variables in Scatter Plot (types of discretiation)

### DISCUSSION THREAD

**Discussion by pepeto** [biolab/orange3#5792] [Ideas]
When a color, from a numeric variable,  is applied on a scatter plot; the widget uses a width discretization. This criterion causes that, in the case of a very popular value range, the plot is all painted with the same color. So there is no added information to the plot.

Is it possible, in future versions, to have the option to choose between equal width and frequency?

THANK YOU!!!! THANK YOU!!!! THANK YOU!!!! THANK YOU!!!! I LOVE ORANGE.

**Comment by ajdapretnar** [comment]
You could try discretizing the variable with equal frequency (Discretize) and perhaps assign sensible colors to the bins so that they would resemble continuous data. But we will think about your proposal, thanks!
